src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1895,7 +1895,7 @@ export default class CanvasController {
     _drawVideoMap(cc) {
         const airportModel = AirportController.airport_get();
 
-        if (!_has(airportModel, 'maps') || !this._shouldDrawVideoMap) {
+        if (!_has(airportModel, 'mapCollection') || !this._shouldDrawVideoMap) {
```

Comment:
we may want to create a method on the `airportModel` to handle this.  basically move this logic there within a method, so you could do: 
```js
if (!airportModel.hasMapCollection()) {
    return;
}
```

Reply:
Makes sense